onyo/lib/tests/test_commands_new.py

```diff
@@ -255,3 +260,18 @@ def test_onyo_new_clones(inventory: Inventory) -> None:
     # equals existing asset except for path and serial:
     assert all(v == new_asset[k] for k, v in existing_asset.items() if k not in ['serial', 'path'])
     assert new_asset['serial'] == 'whatever'
+
+
+@pytest.mark.ui({'yes': True})
+def test_onyo_new_asset_dir(inventory: Inventory) -> None:
```

Comment:
Based on #561, we should add a test with `"is_asset_directory": False`, and possibly `"is_asset_directory": false`, too, that create a normal asset.

I assume that #561 is just happening for the CLI because everything gets interpreted as a string instead of a bool, but to make sure the value to the key is considered correctly is IMO still worth doing. 

What happens (or should happen) for things like `"is_asset_directory": ["this fullfills truthyness requirements"]`? > depends obviously on the check done, but since this is about a reserved key, it is worth considering :)

Reply:
There's a confusion here. What is `"is_asset_directory": false` supposed to be? That's not valid python ;-)

As you suspected, it's only about strings coming in from the CLI and being converted correctly before handed off to us by argparse.

Reply:
(Fixed already)